litmus/libdir/_aarch64/kvm_fault_handler.c

```diff
@@ -16,9 +16,11 @@ static void fault_handler(struct pt_regs *regs,unsigned int esr) {
 }
 
 static void install_fault_handler(int cpu) {
-  install_exception_handler(EL1H_SYNC, ESR_EL1_EC_DABT_EL1,fault_handler);
+  install_exception_handler(EL1H_SYNC, ESR_EL1_EC_DABT_EL1, fault_handler);
+  install_exception_handler(EL1H_SYNC, ESR_EL1_EC_UNKNOWN, fault_handler);
 #ifdef USER_MODE
   struct thread_info *ti = thread_info_sp(user_stack[cpu]);
   ti->exception_handlers[EL0_SYNC_64][ESR_EL1_EC_DABT_EL0] = fault_handler;
+  ti->exception_handlers[EL0_SYNC_64][ESR_EL1_EC_UNKOWN] = fault_handler;
```

Comment:
Sorry there is a typo here...
```suggestion
  ti->exception_handlers[EL0_SYNC_64][ESR_EL1_EC_UNKNOWN] = fault_handler;
```